lib/post_revisor.rb

```diff
@@ -422,7 +426,6 @@ def diff_size(before, after)
 
   def grace_period_edit?
     return false if (@revised_at - @last_version_at) > SiteSetting.editing_grace_period.to_i
-    return false if @post.reviewable_flag.present?
```

Comment:
`#grace_period_edit?` is called in one additional place related to slow mode, but that logic shouldn't take into consideration flags anyway, so this incidentally fixes a minor problem there as well.